python/ray/worker.py

```diff
@@ -976,7 +972,7 @@ def _wait_for_and_process_task(self, task):
             driver_id, function_id.id()) == execution_info.max_calls)
         if reached_max_executions:
             self.local_scheduler_client.disconnect()
-            os._exit(0)
+            sys.exit(0)
```

Comment:
@robertnishihara this doesn't have to be an os._exit right? That prevents exit hooks from running.

Reply:
It doesn't need to be anything in particular. Just needs to exit the process.